csharp/src/Apache.Arrow/Types/Decimal256Type.cs

```diff
@@ -17,14 +17,20 @@ namespace Apache.Arrow.Types
 {
     public sealed class Decimal256Type: FixedSizeBinaryType
     {
+        public static readonly Decimal256Type Default = new(38, 18, 32);
```

Comment:
Maybe a comment here that this corresponds to the built-in C# `decimal` type's max range?

Reply:
Added